pkg/kademlia/routing_test.go

```diff
@@ -177,52 +177,12 @@ func TestConnectionSuccess(t *testing.T) {
 	}
 }
 
-func TestUpdateSelf(t *testing.T) {
```

Comment:
what happens if a storage node's local address changes? i guess we don't currently have that since we configure the external address in the config file, but eventually we may want to handle ip changes automatically

Reply:
i guess i talked myself into that it's okay for now

Reply:
`UpdateSelf` was invoked just to update the available capacity in the `self` node in the own routing table. DHT does not hold this info anymore, so it does not make sense to have `UpdateSelf` anymore. Plus, it caused a data race, which was the main reason to remove it.

I guess updating the address of the `self` node in DHT was and still is an open issue.

Reply:
Well... the data race was not really resolved. So I brought back the `UpdateSelf`, but it now explicitly takes care for updating the capacity in the self node: https://github.com/storj/storj/pull/1785/commits/fd278005671deafbec76f089a9851505b1872a55